Firmware/Filament_sensor.h

```diff
@@ -46,8 +46,10 @@ class Filament_sensor {
 
     static void setEnabled(bool enabled);
 
+    #ifndef REMOVE_AUTOLOAD_FILAMENT_MENU_ENTRY
     void setAutoLoadEnabled(bool state, bool updateEEPROM = false);
     bool getAutoLoadEnabled() const { return autoLoadEnabled; }
+    #endif //NOT REMOVE_AUTOLOAD_FILAMENT_MENU_ENTRY
```

Comment:
I don't think this is the correct approach, you're disabling too much. These functions are not just used for the Autoload menu item. It's used by `triggerFilamentInserted()` too for normal autoload functionality.

Reply:
Actually these shouldn't be used when `autoload` is disabled. There in the 3rd commit I removed even more code and tested it with MK404
- MK3S
- MK3S + MMU
- MK2.5S